docs/contributing/introduction-to-contributing.md

```diff
@@ -17,24 +17,25 @@ keywords:
 ---
 
 Contributions are always welcome, no matter how large or small. Before contributing,
-please read the [code of conduct](./code-of-conduct.md) and follow the directions below:
+please read the [Code of Conduct](./code-of-conduct.md) and follow the directions below:
 
 ## Recommended Communication Style
 
-1. Always leave screenshots for visual changes.
-2. Always leave a detailed description in the pull request. Leave nothing ambiguous for the reviewer.
-3. Always review your code first. Run the project locally and test it before requesting a review.
-4. Always communicate in the GitHub repository. Whether in the issue or the pull request, keeping the lines of communication open and visible to everyone on the team helps everyone around you.
+- Always leave screenshots for visual changes.
+- Always leave a detailed description in the pull request. Leave nothing ambiguous for the reviewer.
+- Always review your code first. Run the project locally and test it before requesting a review.
+- Always communicate in the GitHub repository. Whether in the issue or the pull request, keeping the lines of communication open and visible to everyone on the team helps everyone around you.
 
 ## Issues
 
-If you wish to work on an open issue, please comment on the issue with `.take`, and it will be assigned to you. If an issue is not assigned, it is assumed to be available for anyone to work on. Please assign yourself to an issue before working on it to avoid conflicts.
+- If you wish to work on an open issue, please comment on the issue with `.take`, and it will be assigned to you. If an issue is not assigned, it is assumed to be available for anyone to work on. Please assign yourself to an issue before working on it to avoid conflicts.
+- Issues with the `needs triage` label are unavailable to `.take` until they are triaged and the label is removed. Feel free to check on the issue regularly if you want to work on it.
 
 If you contribute to the project for the first time, you can search for issues with `good first issue` or `bug` labels in the repository. Please only self-assign a `good first issue` one time.
 
 In case you get stuck, feel free to ask for help in the [Discord](https://discord.gg/U2peSNf23P) server or [GitHub Discussions](https://github.com/orgs/open-sauced/discussions).
```

Comment:
These are the paragraphs that need to be transformed into list.

```suggestion
- If you contribute to the project for the first time, you can search for issues with `good first issue` or `bug` labels in the repository. Please only self-assign a `good first issue` one time.
- In case you get stuck, feel free to ask for help in the [Discord](https://discord.gg/U2peSNf23P) server or [GitHub Discussions](https://github.com/orgs/open-sauced/discussions).
```

Reply:
@HappyLife2, here's a guide on how to do lists in Markdown: https://www.markdownguide.org/basic-syntax/#lists-1